lib/validator.js

```diff
@@ -281,6 +282,15 @@ class Validator {
 				schema.optional = true;
 		}
 
+		const alias = this.aliases[schema.type];
+
+		if (alias) {
+			schema.type = alias.type;
+			delete schema.type;
+			Object.assign(alias, schema);
```

Comment:
It doesn't correct because it overwrites the original alias. So if you use it multiple times it won't be the same.
Use `Object.assign({}, alias, schema)` to create a new instance.